src/test_workflow/smoke_test/smoke_test_runner_opensearch.py

```diff
@@ -72,8 +72,11 @@ def start_test(self, work_dir: Path) -> TestSuiteResults:
                             response = requests_method(request_url, verify=False, auth=("admin", "myStrongPassword123!"), headers=header, data=parameters_data)
                             logging.info(f"Response is {response.text}")
                             self.validate_response_swagger(response)
-                        except:
+                        except Exception as e:
                             status = 1
+                            logging.error(f"Unexpected Error type is {type(e)}")
+                            logging.error(e)
```

Comment:
Why do we need to know the type of error? Also do we need to rethrow it properly?

Reply:
I think it would be helpful to know the type of the exception to get more contexts on what's failing. For example, there are `PathsNotFound`, `ServerNotFound`, `ResponseNotFound` and etc... 
Though each exception will output the error text, I believe it's no harm to know the type of it for debugging purpose. 
I don't have strong opinion to keep it or not.